### PR TITLE
Setup coveralls

### PR DESCRIPTION
**Pull request by radiofreemattd**
This includes the configuration that should, upon a successful build, push code coverage statistics to coveralls.io - as of now, I have commented out a rake task in the deployment settings because it cannot find any such named task in master (because this has not yet been merged). Once merged, I think we can un-comment that out and everything should work.

Edit - after talking to @jonmast, I don't think we need the Rake task, which I have now deleted...

### CODE REVIEW

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
-require 'simplecov'
-SimpleCov.start
+require 'coveralls'
+Coveralls.wear!
```
Comment (jonmast) [inline spec/rails_helper.rb]: I checked coveralls' docs and it looks like they say they recommend `Coveralls.wear!('rails')` for rails projects. My apologies if I'm misunderstanding something.

File: lib/tasks/coveralls.rake [hunk 2]
```diff
@@ -0,0 +1,2 @@
+require 'coveralls/rake/task'
+Coveralls::RakeTask.new
```
Comment (jonmast) [inline lib/tasks/coveralls.rake]: Is there a specific reason you included this? From my reading of the docs it shouldn't be necessary.

**Comment by jonmast** [comment]
Looks Great :+1:

File: Gemfile [hunk 3]
```diff
@@ -58,11 +58,12 @@ group :development, :test do
 end
 
 group :test do
-  # Use Caoybara for acceotance testing
+  # Use Capybara for acceotance testing
```
Comment (mscoutermarsh) [inline Gemfile]: typo "acceotance"

**Comment by mscoutermarsh** [comment]
looks good! merge whenevaa :+1:

**Pull request merged**